### PR TITLE
Update Slinky module

### PR DESCRIPTION
**Pull request by Marshall-Hallenbeck**

### CODE REVIEW

**Comment by Marshall-Hallenbeck** [comment]
I really want to fix #256 for this, because I want to be able to log what files were created/deleted to a file, so that users can reference it for IOC/cleanup, because right now you have to log all your output, and even then it's a bit messy (sometimes stuff fails to delete, etc).

File: nxc/protocols/smb.py [hunk 1]
```diff
@@ -76,6 +76,7 @@
     "STATUS_INVALID_LOGON_HOURS",
     "STATUS_INVALID_WORKSTATION",
     "STATUS_LOGON_TYPE_NOT_GRANTED",
+    "STATUS_LOGON_FAILURE",
```
[the remark this answers is not in the document]
Comment (Marshall-Hallenbeck) [reply nxc/protocols/smb.py:79]: > Be carrefull, this simple addition breaks all the "purple" / "red" information on the smb proto. Now every accounts (even the one that doesn't exist) should be printed with a purple mark which is incorrect

Hmm how does it break that?

This error was being thrown during connections, so I added it in here. I can remove it though.

[the remark this answers is not in the document]
Comment (mpgn) [reply nxc/protocols/smb.py:79]: https://github.com/Pennyw0rth/NetExec/blob/main/nxc/protocols/smb.py#L406 :)

**Review by mpgn — APPROVED** [review]

**Comment by Marshall-Hallenbeck** [comment]
I think there might be one more thing I gotta fix but it's hard to test. I'll create another PR for that.

**Pull request merged**